crates/serpen/tests/fixtures/ast_rewriting/variable_name_collision/main.py

```
@@ -0,0 +1,25 @@
+
+
+def main():
+    # Use variables from both modules
+    print(f"Constants API URL: {API_URL}")
+    print(f"Config API URL: {CONFIG_API_URL}")
+    print(f"Constants Version: {VERSION}")
+    print(f"Config Version: {CONFIG_VERSION}")
+    print(f"Default Timeout: {DEFAULT_TIMEOUT}")
+    print(f"Max Retries: {MAX_RETRIES}")
+
+    return {"constants_api_url": API_URL, "config_api_url": CONFIG_API_URL, "constants_version": VERSION, "config_version": CONFIG_VERSION, "default_timeout": DEFAULT_TIMEOUT, "max_retries": MAX_RETRIES}
```

Pylint (3.3.7)

[convention] Line too long (203/100)

(C0301)